openapi-spec/swagger.yaml

```diff
@@ -521,7 +522,7 @@ paths:
         - name: body
           in: body
           schema:
-            $ref: '#/definitions/ExtendVolumeSpec'
+            $ref: '#/definitions/ExtendVolume'
```

Comment:
API says "resize", but description and schema says "extend".  Not consistent. I'd prefer API is changed to "extend".  Just reminded me of this.  You don't need to change anything as part of this PR.

Reply:
Got it, I will assign this issue to @BaiHuoYu